### PR TITLE
Dryoutformsto editmultipleuser #169

### PR DESCRIPTION
**Pull request by vijayakarumudi**
Closes AgileVentures/MetPlus_tracker#169.

Dry out forms for jobseeker new/edit, company-person edit and Agency-person edit.

### CODE REVIEW

**Comment by vijayakarumudi** [comment]
close the PR

**Comment by vijayakarumudi** [comment]
PR for review

File: db/seeds.rb [hunk 1]
```diff
@@ -74,18 +74,6 @@
   end
 end
 
-#SkillLevel
-SkillLevel.create(name: 'Beginner',
```
Comment (joaopapereira) [inline db/seeds.rb]: We don't need the skill level anymore?

**Comment by joaopapereira** [comment]
The change looks good, please review my previous comment

File: db/seeds.rb (hunk 1 above)
Comment (vijayakarumudi) [inline db/seeds.rb]: Skill level table was already dropped from ERD. I had merge conflicts with seed file. Patrick and I did  PP session. He cleaned up the seed file.

**Comment by joaopapereira** [comment]
Going to merge

**Pull request merged**